classes/AdminPayPalController.php

```diff
@@ -214,16 +214,20 @@ protected function _checkTLSVersion()
             'error_message' => '',
         ];
 
-        if (defined('CURL_SSLVERSION_TLSv1_2') == false) {
-            define('CURL_SSLVERSION_TLSv1_2', 6);
+        if (defined('CURL_SSLVERSION_TLSv1_3')) {
+            $tlsVersion = CURL_SSLVERSION_TLSv1_3;
+        } elseif (defined('CURL_SSLVERSION_TLSv1_3')) {
```

Comment:
I am not sure to undestand this condition.

Perhaps it should be : 'CURL_SSLVERSION_TLSv1_2' ?

Because if is defined 'CURL_SSLVERSION_TLSv1_3' and not defined 'CURL_SSLVERSION_TLSv1_2' we could have an error don't you think ?

And we will have 1.2 enabled if both 1.3 and 1.2 is defined as constant.

Reply:
Fixed,
Thanks